### ISSUE TITLE
The email always contain all the entries instead of recently added

### ISSUE THREAD

**Issue by aksharj** [ties/logcheck-extrarules#11]
Hi All,

Sorry if this is not the correct place to report this but i am stuck and not able to find a solution.

I have configured logcheck but when evrytime when cron runs the email that gets sent contains log lines from the start i.e. from line 0.

Is there any configuration i need to make so that the email that gets triggered only contains new log lines.

i have the below rule in the violations.d/trusted

```
.*(could).*$
```

when the cron is triggered i get the log lines from 0, i would expect the email to only contain latest lines which are not already processed.

Please help. How do i ensure that i only get new lines reported.

@ties tagging for help.

Thanks.

**Comment by ties** [comment]
Hi,

I'm not sure how to configure logcheck to only send the new lines since the last incident. That should be the default.

However I would put your rule in `violations-ignore.d` if you do not want to get notified about a line matching it. You can also try `logcheck-test` for debugging.

**Comment by aksharj** [comment]
Thanks @ties , i integrated logcheck and see that email does only contain the new entries.
